DeleteUserDataActivity/backupAndDelete.ts

```diff
@@ -155,13 +203,17 @@ const backupAndDeleteProfile = ({
   fiscalCode,
   profileModel,
   servicePreferencesModel,
-  userDataBackup
+  userDataBackup,
+  profileEmailsRepository,
+  lastValidatedEmail
 }: {
   readonly authenticationLockService: AuthenticationLockService;
   readonly profileModel: ProfileDeletableModel;
   readonly userDataBackup: IBlobServiceInfo;
   readonly servicePreferencesModel: ServicePreferencesDeletableModel;
   readonly fiscalCode: FiscalCode;
+  readonly profileEmailsRepository: IProfileEmailWriter;
+  readonly lastValidatedEmail?: EmailString;
```

Comment:
Can we use `O.Option<EmailString>` instead?

Reply:
it's better not to use `Option` in parameters. It's easier to access the argument and so it can be easily validated

Reply:
Since `getLastEmailIfValidated` is returning an Option, we can avoid useless steps [here](https://github.com/pagopa/io-functions-admin/blob/84c3516a61eca6832ac3c4a721ad34b42e42225f/DeleteUserDataActivity/backupAndDelete.ts#L693)

Reply:
I put `getLastEmailIfValidated` into `backupAndDeleteProfile` as you preferred so I removed `lastValidatedEmail` from the parameters